patterns/hero-full-width-image.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Title: Hero Full Width Image
+ * Slug: twentytwentyfive/hero-full-width-image
+ * Categories: banner, featured
+ * Description: A hero with a full width image, heading, short paragraph and button
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:cover {"url":"<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/image-from-rawpixel-id-8802603-original.webp","dimRatio":20,"overlayColor":"black","isUserOverlayColor":true,"focalPoint":{"x":0.5,"y":0.95},"minHeight":832,"minHeightUnit":"px","contentPosition":"bottom left","align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|50","bottom":"var:preset|spacing|50","left":"var:preset|spacing|50","right":"var:preset|spacing|50"}}},"layout":{"type":"constrained"}} -->
```

Comment:
could you include an image license in the readme.txt file?

Reply:
The block spacing for the elements in the cover appears to be `30px`.

https://github.com/user-attachments/assets/cbf46d7d-39fe-4dd1-b75a-e88491badade

This can be probably solved by setting the block spacing for the cover, to "Small".